src/Microsoft.DocAsCode.Build.ManagedReference/ManagedReferenceDocumentProcessor.cs

```diff
@@ -209,6 +209,7 @@ public virtual FileModel LoadIntermediateModel(Stream stream)
         protected virtual void UpdateModelContent(FileModel model)
         {
             var apiModel = ApiBuildOutput.FromModel((PageViewModel)model.Content); // Fill in details
+            var serialzied = JsonUtility.Serialize(apiModel, Formatting.Indented);
```

Comment:
oops


---
In reply to: [102859648](https://github.com/dotnet/docfx/pull/1341#discussion_r102859648) [](ancestors = 102859648)